omniledger/service/service.go

```diff
@@ -191,6 +195,44 @@ func (s *Service) AddTransaction(req *AddTxRequest) (*AddTxResponse, error) {
 
 	s.txBuffer.add(string(req.SkipchainID), req.Transaction)
 
+	if req.InclusionWait != nil {
```

Comment:
This is different from the `InclusionWait` documentation in `proto.go`. The documentation says if it's 0, then do nothing. But here it's check whether it's unset. Maybe use `req.InclusionWait != nil || req.InclusionWait > 0`. Alternatively, we can just change `InclusionWait` to a non-optional field, which defaults to 0 if not set. I prefer the latter a bit more because it as fewer moving parts.

Reply:
In fact the `dedis/protobuf` library handles pointers as optional parameters. But perhaps I should start using 'optional' in there...

Reply:
changed `proto.awk` to accept 
```
`protobuf="opt"`
```
as optional parameter.